test/unit/tasks/push/creation.test.js

```diff
@@ -188,6 +188,115 @@ test('Create entries and handle regular errors', () => {
     })
 })
 
+test('Create private tags', () => {
```

Comment:
Can the visibility value in the original be different from the visibility in the transformed data?

Reply:
No, the transformer just returns `tag` as it came in